packages/storybook/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@department-of-veterans-affairs/storybook",
+  "packageManager": "yarn@3.0.2",
+  "scripts": {
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
+  },
+  "dependencies": {
+    "@department-of-veterans-affairs/component-library": "workspace:packages/core",
```

Comment:
`storybook` is its own package since it contains stories for `react-components` _and_ `web-components`. Leaving it in `react-components` didn't make much sense now that we have a monorepo.